docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -0,0 +1,872 @@
+[role="xpack"]
+[[ootb-ml-jobs]]
+== Prebuilt {anomaly-jobs}
+
+{anomaly-jobs-cap} contain the configuration information and metadata necessary 
+to perform an analytics task. {kib} can recognize certain types of data and 
+provide specialized wizards for that context. This page lists the available 
+prebuilt {anomaly-jobs} that you can use via {kib}. Every module item in the 
+list contains a name, a short description, information about 
+the {ml-docs}/ml-buckets.html#ml-buckets[`bucket_span`] and the 
+{ml-docs}/ml-functions.html[analysis function] as well as the influencers – the 
+entities that have contributed to, or are to blame for, the anomalies – of the 
+{anomaly-detect} job.
+
+
+[float]
+[[ootb-ml-jobs-apache]]
+=== Apache
+
+These {anomaly-job} wizards appear in {kib} if you use 
+{filebeat-ref}/index.html[{filebeat}] to ship access logs from your 
+https://httpd.apache.org/[Apache] HTTP servers to {es} and store it using fields 
+and datatypes from the Elastic Common Schema (ECS).
+
+low_request_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `apache.access`.
+* Models the event rate of HTTP requests. 
+* Detects unusually low counts of HTTP requests compared to the previous event 
+rate.
+
+This job can be created if web access log data exists matching the defined 
```

Comment:
> This job can be created if...

This sentence seems to duplicate the first item in the list, so I'd recommend omitting it.